papergres.go

```diff
@@ -436,89 +447,90 @@ func (s *Schema) InsertAll(objs interface{}) ([]*Result, error) {
 		return nil, err
 	}
 	if len(slice) == 0 {
-		return nil, errors.New("Empty slice")
+		return nil, errors.New("empty slice")
 	}
 
-	// sql, args := insertMultipleSQL(objs, s.Name)
-	// return s.Database.Query(sql, args...).ExecNonQuery(), nil
-
 	// now turn the objs into a repeat query and exec
 	return s.GenerateInsert(slice[0]).Repeat(len(slice),
 		func(i int) (dest interface{}, args []interface{}) {
-			args = insertArgs(slice[i])
+			args = insertArgs(slice[i], false)
 			return
 		}).Exec()
 }
 
+// GenerateInsert generates the insert query for the given object
+func (s *Schema) GenerateInsert(obj interface{}) *Query {
+	return s.generateInsertQuery(obj, false)
+}
+
+// GenerateInsertWithPK generates the insert query for the given object in which
```

Comment:
See above regarding comments.

Reply:
Same as above. Line comments are the norm.